Add Yoto sensors documentation (#45810)
home-assistant/home-assistant.github.io: source/_integrations/yoto.markdown

ha_domain: yoto
ha_platforms:
  - media_player
  - sensor
ha_integration_type: hub
ha_dhcp: true
---

> The **Yoto** {% term integration %} lets you control your [Yoto](https://yotoplay.com) audio players from Home Assistant. You can play and pause cards, change the volume, skip tracks, seek within a track, see what is currently playing, and browse your card library to start a specific card, chapter, or track. You can also monitor each player's battery level, what is loaded in the card slot, and its Wi-Fi connection.

Reviewer: ```suggestion
The **Yoto** {% term integration %} lets you control your [Yoto](https://yotoplay.com) audio players from Home Assistant. You can play and pause cards, change the volume, skip tracks, seek within a track, see what is currently playing, and browse your card library to start a specific card, chapter, or track. You can also monitor each player's battery level, what is loaded in the card slot, and its Wi-Fi signal strength.
```

  media_content_id: "yoto://card/abc123/01/02"
```

### Sensors

Each Yoto player also provides several sensors:

- **Battery**: the player's battery charge.
- **Card slot**: what is loaded in the player, such as a physical card or streaming content.

> - **Day mode**: whether the player is in day or night mode.

Reviewer: ```suggestion
- **Day mode**: the player's current mode (day or night).
```
apply parallelism
